### PR TITLE
Updated go version to 1.17.3

### PR DESCRIPTION
**Pull request by pdhanoyasplunk**

### CODE REVIEW

File: env [hunk 1]
```diff
@@ -0,0 +1 @@
+CLUSTER_WORKERS=5
```
Comment (smohan-splunk) [inline env:1]: @pdhanoyasplunk Can you please clarify if this change is intended?

Comment (pdhanoyasplunk) [reply env]: Thanks for catching. Removed.

**Review by smohan-splunk — APPROVED** [review]

**Review by tpereirasplunk — APPROVED** [review]

**Review by jambrosiano — APPROVED** [review]

**Review by kashok-splunk — APPROVED** [review]

**Review by gaurav-splunk — APPROVED** [review]

**Comment by sgontla** [comment]
Since we are moving to the new go version, are we going to update the docs as well?

**Comment by smohan-splunk** [comment]
> Since we are moving to the new go version, are we going to update the docs as well?
@sgontla Good point, CSPL-1455 created to track this

**Pull request merged**